src/commands/force/package/beta/installed/list.ts

```diff
@@ -5,22 +5,54 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import { Messages, SfdxPropertyKeys } from '@salesforce/core';
+import { Messages } from '@salesforce/core';
 import { SfdxCommand } from '@salesforce/command';
+import { packageInstalledList } from '@salesforce/packaging';
+import { CliUx } from '@oclif/core';
 
 Messages.importMessagesDirectory(__dirname);
 const messages = Messages.loadMessages('@salesforce/plugin-packaging', 'package_installed_list');
 
+export type PackageInstalledListResult = {
+  Id: string;
+  SubscriberPackageId: string;
+  SubscriberPackageName: string;
+  SubscriberPackageNamespace: string;
+  SubscriberPackageVersionId: string;
+  SubscriberPackageVersionName: string;
+  SubscriberPackageVersionNumber: string;
+};
+
 export class PackageInstalledListCommand extends SfdxCommand {
   public static readonly description = messages.getMessage('cliDescription');
   public static readonly longDescription = messages.getMessage('cliLongDescription');
   public static readonly help = messages.getMessage('help');
-  public static readonly orgType = SfdxPropertyKeys.DEFAULT_USERNAME;
   public static readonly requiresUsername = true;
-  public static readonly requiresProject = true;
 
-  public async run(): Promise<unknown> {
-    process.exitCode = 1;
-    return Promise.resolve('Not yet implemented');
+  public async run(): Promise<PackageInstalledListResult[]> {
+    const result = await packageInstalledList(this.org.getConnection());
+
+    const records = result.map((record) => ({
+      Id: record.Id,
+      SubscriberPackageId: record.SubscriberPackageId,
+      SubscriberPackageName: record.SubscriberPackage.Name,
+      SubscriberPackageNamespace: record.SubscriberPackage.NamespacePrefix,
+      SubscriberPackageVersionId: record.SubscriberPackageVersion.Id,
+      SubscriberPackageVersionName: record.SubscriberPackageVersion.Name,
+      SubscriberPackageVersionNumber: `${record.SubscriberPackageVersion.MajorVersion}.${record.SubscriberPackageVersion.MinorVersion}.${record.SubscriberPackageVersion.PatchVersion}.${record.SubscriberPackageVersion.BuildNumber}`,
+    }));
+
+    const tableOptions: CliUx.Table.table.Options = {
+      Id: { header: 'ID' },
+      SubscriberPackageId: { header: 'Package ID' },
+      SubscriberPackageName: { header: 'Package Name' },
+      SubscriberPackageNamespace: { header: 'Namespace' },
+      SubscriberPackageVersionId: { header: 'Package Version ID' },
+      SubscriberPackageVersionName: { header: 'Version Name' },
+      SubscriberPackageVersionNumber: { header: 'Version' },
+    };
+    this.ux.table(records, tableOptions);
```

Comment:
3rd property keeps the table from being obnoxious.  Seems oclif uses it true by default
```suggestion
    this.ux.table(records, tableOptions, { 'no-truncate': false });
```